README.taint

```diff
@@ -0,0 +1,143 @@
+Taint Suport for Jim Tcl
+========================
+
+Author: Steve Bennett <steveb@workware.net.au>
+Date: 24 May 2011
```

Comment:
Was this really planned 10 years ago?

Reply:
Yes, really. Been out of tree for that long. I thought it was time to share.
Thanks for your edits below. I'll integrate them.